pkgs/applications/misc/bottles/fhsenv.nix

```diff
@@ -0,0 +1,101 @@
+{ lib
+, buildFHSUserEnv
+, symlinkJoin
+, bottles-unwrapped
+, extraPkgs ? pkgs: [ ]
+, extraLibraries ? pkgs: [ ]
+}:
+
+let fhsEnv = {
+  targetPkgs = pkgs: with pkgs; [
+    bottles-unwrapped
+    vkbasalt
```

Comment:
So just by having it in the fhsEnv like this doesn't work?

Reply:
It is necessary to have `vkbasalt` in `targetPkgs` because Bottles doesn't let you enable the checkbox if it can't find vkBasalt in (you guessed it) a hardcoded path.

This is basically like patching Bottles to force "checkbox enableable" to True (to be fair, I could do that instead (it achieves the same result), but it would be harder to maintain). As I said, the checkbox won't do anything if `vkbasalt` isn't in `environment.systemPackages` (or `nix-env -i`).

If I don't put `vkbasalt` in `targetPkgs` (or patch Bottles), then it would be "the checkbox is forced off even if `vkbasalt` is in `environment.systemPackages`".